[Arch with Chaotic-AUR] unittest in TestTools::test_filesystem & test_device do fail on Chaotic-AUR build system because of overlayfs

The same failure happens for [backintime](https://aur.archlinux.org/packages/backintime) & [backintime-git](https://aur.archlinux.org/packages/backintime-git). Might just a need a check for [`overlayfs`](https://wiki.archlinux.org/title/Overlay_filesystem).

Build logs:
[backintime-git.log](https://builds.garudalinux.org/repos/chaotic-aur/logs/backintime-git.log)
[backintime.log](https://builds.garudalinux.org/repos/chaotic-aur/logs/backintime.log)

```py
python -m unittest  -b test/test_tools.py
.....s..F.Fs.s..s......s................s..s.s...ss.s.s.............
======================================================================
FAIL: test_device (test.test_tools.TestTools)
----------------------------------------------------------------------
Traceback (most recent call last):
File "/home/main-builder/pkgwork/src/backintime-1.3.2/common/test/test_tools.py", line 447, in test_device
self.assertRegex(tools.device('/nonExistingFolder/foo/bar'),
AssertionError: Regex didn't match: '(:?/dev/.*|tmpfs)' not found in 'overlay'

======================================================================
FAIL: test_filesystem (test.test_tools.TestTools)
----------------------------------------------------------------------
Traceback (most recent call last):
File "/home/main-builder/pkgwork/src/backintime-1.3.2/common/test/test_tools.py", line 453, in test_filesystem
self.assertRegex(tools.filesystem('/nonExistingFolder/foo/bar').lower(),
AssertionError: Regex didn't match: '(:?ext[2-4]|xfs|zfs|jfs|raiserfs|btrfs|tmpfs)' not found in 'overlay'
```
This was found thanks to a request to add both packages to Chaotic-AUR in https://github.com/chaotic-aur/packages/issues/1733

--- a/common/test/test_tools.py
+++ b/common/test/test_tools.py
@@ -435,15 +435,16 @@ class TestTools(generic.TestCase):
     def test_device(self):
         self.assertEqual(tools.device('/proc'), 'proc')
         self.assertRegex(tools.device('/sys'), r'sys.*')
-        self.assertRegex(tools.device('/nonExistingFolder/foo/bar'),
-                         r'(:?/dev/.*|tmpfs|instances/containers/travis.*)')
+        self.assertRegex(
+            tools.device('/nonExistingFolder/foo/bar'),
+            r'(:?/dev/.*|tmpfs|overlay|instances/containers/travis.*)')
 
     def test_filesystem(self):
         self.assertEqual(tools.filesystem('/proc'), 'proc')
         self.assertRegex(tools.filesystem('/sys'), r'sys.*')
         self.assertRegex(
             tools.filesystem('/nonExistingFolder/foo/bar').lower(),
-            r'(:?ext[2-4]|xfs|zfs|jfs|raiserfs|btrfs|tmpfs|shiftfs)')
+            r'(:?ext[2-4]|xfs|zfs|jfs|raiserfs|btrfs|tmpfs|overlay|shiftfs)')
 
     # tools.uuidFromDev() get called from tools.uuidFromPath because the
     # latter is a synonym/surrogate for too.suuidFromDev()
@@ -479,14 +480,17 @@ class TestTools(generic.TestCase):
         # dev-disk folder
         path_dev = pathlib.Path('/dev')
         fake_fs.create_dir(path_dev)
-        
+
         # create disk-files from "sda1" to "sda4" to "sdd4"
-        dev_list = [path_dev / f'sd{letter}{number}' for letter in list('abcd') for number in range(1, 5)]
+        dev_list = [
+            path_dev / f'sd{letter}{number}'
+            for letter in list('abcd')
+            for number in range(1, 5)]
 
         # dev-disk-by-uuid
         path_by_uuid = pathlib.Path('/dev') / 'disk' / 'by-uuid'
         fake_fs.create_dir(path_by_uuid)
-        
+
         # uuids
         uuid_list = [str(uuid.uuid4()) for _ in range(16)]
 
@@ -494,7 +498,7 @@ class TestTools(generic.TestCase):
         for idx in range(16):  # 16 devices
             fake_fs.create_symlink(
                 # e.g. /dev/disk/by-uuid/c7aca0
-                file_path=path_by_uuid / uuid_list[idx],  
+                file_path=path_by_uuid / uuid_list[idx],
                 # e.g. /dev/sda1
                 link_target=path_dev / dev_list[idx]
             )

